atmo/clusters/provisioners.py

```diff
@@ -39,7 +39,10 @@ def start(self, user_email, identifier, emr_release, size, public_key):
                 'Name': 'setup-telemetry-cluster',
                 'ScriptBootstrapAction': {
                     'Path': self.script_uri,
-                    'Args': ['--public-key', public_key]
+                    'Args': [
+                        '--public-key', public_key,
+                        '--email', user_email,
```

Comment:
@fbertsch Can you check if the values (see settings.py in this PR) are still the correct ones? Once in product we'll be able to update those values without deploys (update via Django admin).